src/MizarBase/include/MizarBase/Address.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_BASE_ADDRESS_H_
+#define MIZAR_BASE_ADDRESS_H_
+
+#include <functional>
+
+#include "OrbitBase/Typedef.h"
+
+namespace orbit_mizar_base {
+
+struct AddressTag {};
+using Address = orbit_base::Typedef<AddressTag, const uint64_t>;
```

Comment:
The name must be more descriptive. What kind of address is this used for?
Absolute address (from sampling)? -- I guess so.
Or are those relative addresses?

If we introduce strong types here, it is important to take the chance to not mix those kind of addresses.

Reply:
I see your point. But this is Mizar-specific type (at least, for now). We don't have relative addresses here. Wouldn't it make sense not to rename at least the aliases?

Reply:
I would be fine with not renaming the alias. But what would be your point?

Reply:
Using a more descriptive name is supposed to resolve ambiguity (absolute vs relative address), which does not exist in Mizar. So the longer type name wouldn't help here.

Reply:
On the second thought, `using Address = ::blah_blah::AbsoluteAddress` looks strange. I'll rename everything.